### PR TITLE
Remove use of cmdlet aliases from .\test\powershell

### PR DESCRIPTION
**Pull request by xtqqczze**
## PR Summary

* Remove use of cmdlet aliases from .\test\powershell directory.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: test/powershell/Language/Scripting/CheckRestrictedlanguage.Tests.ps1 [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSAvoidUsingCmdletAliases", "")]
+param()
```
Comment (iSazonov) [inline test/powershell/Language/Scripting/CheckRestrictedlanguage.Tests.ps1]: Please remove these lines.

**Comment by iSazonov** [comment]
@xtqqczze Sorry, I meant that comments is not needed too. We don't have `strong CI checks` for scripts. So such suppressions and related comments makes no sense.

**Comment by xtqqczze** [comment]
@iSazonov When using vscode with the ms-vscode.powershell extension enabled, rule violations from PSScriptAnalyzer testing are listed as warnings in the 'Problems' pane, for example:
```
{
	"resource": "~/test/powershell/Modules/Microsoft.PowerShell.Utility/Set-Variable.Tests.ps1",
	"owner": "_generated_diagnostic_collection_name_#2",
	"code": "PSAvoidUsingCmdletAliases",
	"severity": 4,
	"message": "'sv' is an alias of 'Set-Variable'. Alias can introduce possible problems and make scripts hard to maintain. Please consider changing alias to its full content.",
	"source": "PSScriptAnalyzer",
	"startLineNumber": 178,
	"startColumn": 2,
	"endLineNumber": 178,
	"endColumn": 4
}
```

Certain tests have comments explaining a rule should be ignored, because at the present time rules can only be suppressed on a file level for Pester tests. These comments help identify spurious warnings in the vscode 'Problems' pane and provide a placeholder for future implementation of rule suppression at a test level.

**Comment by iSazonov** [comment]
@xtqqczze These comments is obvious and don't add value but reduce readability. Your proposal will be useful after we (1) get implementation of local suppressions and (2) force CIs. Today I suggest split replacing aliases and suppressing messages. For second work you could open a tracking issue if you want.

**Review by iSazonov — APPROVED** [review]

**Review by SteveL-MSFT — APPROVED** [review]

**Comment by iSazonov** [comment]
@xtqqczze Thanks for your contribution!

**Pull request merged**